compute_tools/src/compute.rs

```diff
@@ -355,6 +357,48 @@ impl ComputeNode {
         Ok(())
     }
 
+    // We could've wrapped this around `pg_ctl reload`, but right now we don't use
+    // `pg_ctl` for start / stop, so this just seems much easier to do as we already
+    // have opened connection to Postgres and superuser access.
+    #[instrument(skip(self, client))]
+    fn pg_reload_conf(&self, client: &mut Client) -> Result<()> {
+        client.simple_query("SELECT pg_reload_conf()")?;
+        Ok(())
+    }
+
```

Comment:
Could also just send SIGHUP to postgres. We know its PID (or if we don't store it anywhere currently, we could).

It's true that this we need a Postgres connection anyway to sync the roles/databases etc. But it would be nice if we could apply postgresql.conf changes anyway, even if all connections are in use or something. Then again, maybe it's better to not apply any of the changes in that case. Not sure...

Reply:
I can say that in autoscaling the config is reloaded in the same manner, cc @sharnoff

Reply:
Yeah, I see 3 options:
- use pg_reload_conf()
- send signal directly from rust. IIUC, this requires usage of unsafe block and libc directly, which is likely fine
- wrap around pg_ctl

I don't really have a preference, the first one worked well so far, so I'd keep it for now